### PR TITLE
README.md: Create

### PR DESCRIPTION
**Pull request by m4tr1k**
Closes https://github.com/coala/community/issues/10

### CODE REVIEW

**Comment by blazeu** [comment]
Why did you close the first one?

It is possible to rewrite the commit there if that's what you want, ask away in our chatroom ;)

**Comment by prnvdixit** [comment]
Wait for someone else to comment too before making any further changes :wink:.

**Comment by jayvdb** [comment]
See the task for why the second PR.
I think a little more creativity is required here, as you have only copied what was in the issue.

Also, you will need to rebase your patch.  Lets conquer this!

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
 # community
+
+![coala Header](/docs/_static/images/coala-header.png)
+
+## About
+
+* This repository is a static django website;
```
Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

Comment (m4tr1k) [reply README.md]: you mean [coala Header]? You want to remove the word coala?

Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on 2bf3493739c9ab53f7bef5705fc4de1d1446ef62, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on 2bf3493739c9ab53f7bef5705fc4de1d1446ef62, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

File: README.md (hunk 1 above)
Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
 # community
+
+![Header](/docs/_static/images/coala-header.png)
+
```
Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on 2bf3493739c9ab53f7bef5705fc4de1d1446ef62, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on 2bf3493739c9ab53f7bef5705fc4de1d1446ef62, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on a09ae1bb708dbf301e45a30ee0b6da11c839d47c, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

File: README.md (hunk 2 above)
Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by jayvdb** [comment]
Comment on a09ae1bb708dbf301e45a30ee0b6da11c839d47c, file README.md, line 3.

The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

File: README.md (hunk 1 above)
Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

Comment (jayvdb) [inline README.md]: The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*

**Comment by nkprince007** [comment]
ack https://github.com/coala/community/pull/23/commits/6973809e1ddeb0354381477cedf8e2fbec6df0e9

**Comment by nkprince007** [comment]
It looks good overall, but since you basically copied the info from the issue. **kudos-**
Would've be better, if you've been a little more creative.

**Pull request merged**